### DISCUSSION TITLE
Add Dataswift.io as a auth provider

### DISCUSSION THREAD

**Discussion by pperzyna** [nuxt-community/auth-module#1480] [Ideas & Feature Requests]
### Is your feature request related to a problem? Please describe.
I would like to use the Dataswift.io solution directly in nuxt-auth

### Describe the solution you'd like to see
I would like to see Dataswift auth in the same way as auth0

### Describe alternatives you've considered
Create a separate module for Dataswift auth? 

### Additional context
Dataswift is an out of box solution where you can store your client's data fully in all compliance with GDPR and clients have always the right to see and delete their data.
https://www.youtube.com/watch?v=vYef5SHr08s&ab_channel=DataswiftforDevelopers

**Comment by bmulholland** [comment]
A PR would be welcome!
